Nova/WinForms/RaceDesigner/RaceDesigner.cs

```diff
@@ -184,7 +184,7 @@ private void OnLoad(object sender, EventArgs e)
             this.iconIndex.Text = Path.GetFileNameWithoutExtension(this.currentRaceIcon.Source);
 
             // Can't trust the windows designer generate code to set the environment range before setting the environment value, so set it here to be sure.
-            this.temperatureTolerance.MinimumValue = 15;
+            this.temperatureTolerance.MinimumValue = 15;            

```

Comment:
I don't understand what's changed here - you added two tabs? And that fixes the graphics?

Reply:
wrong file comitted originally. Actual fix now submitted.